src/envoy/auth/jwt.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PROXY_JWT_H
+#define PROXY_JWT_H
+
+#include "openssl/evp.h"
+#include "rapidjson/document.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace Envoy {
+namespace Http {
+namespace Auth {
+
+class Jwt {
+ public:
```

Comment:
Yes they are.
Just because at first I passed the public key to Jwt class object as an argument of its constructor. I'll make it static.